ui/src/components/LoginForm/SelectProposal.jsx

```diff
@@ -1,182 +1,123 @@
-        </Modal.Header>
-        <Modal.Body>
-          <Form.Control
-            type="text"
-            id="search_session"
-            placeholder="Search"
-            value={this.state.filter}
-            onChange={this.handleChange.bind(this)}
-          />
-          <br />
-
-          <Tabs defaultActiveKey="scheduled" id="scheduled-tab">
-            <Tab
-              eventKey="scheduled"
-              title={`Scheduled (${scheduledSessions.length})`}
-            >
-              <div style={{ overflow: 'auto', height: '550px', padding: 10 }}>
-                <SessionTable
-                  sessions={scheduledSessions}
-                  selectedSessionId={this.state.pId}
-                  filter={this.state.filter}
-                  params={{ showBeamline: false }}
-                  onSessionSelected={this.handleOnSessionSelected}
-                />
-              </div>
-            </Tab>
-            <Tab
-              eventKey="non-scheduled"
-              title={`Non scheduled (${nonScheduledSessions.length})`}
-            >
-              <div style={{ overflow: 'auto', height: '550px', padding: 10 }}>
-                <SessionTable
-                  sessions={nonScheduledSessions}
-                  selectedSessionId={this.state.pId}
-                  filter={this.state.filter}
-                  params={{ showBeamline: true }}
-                  onSessionSelected={this.handleOnSessionSelected}
-                />
-              </div>
-            </Tab>
-          </Tabs>
-        </Modal.Body>
-        <Modal.Footer>
-          {session &&
-            session.is_scheduled_beamline === true &&
-            session.is_scheduled_time === false && (
-              <Button
-                variant="warning"
-                className="float-end"
-                disabled={this.state.pNumber === null}
-                onClick={this.handleSelectProposal}
-              >
-                Reschedule
-              </Button>
-            )}
-          {session && session.is_scheduled_beamline === false && (
-            <Button
-              variant="danger"
-              className="float-end"
-              disabled // {this.state.pNumber === null}
-              onClick={this.handleSelectProposal}
-            >
-              Move here
-            </Button>
-          )}
-          <Button variant="outline-secondary" onClick={this.handleCancel}>
-            Cancel
-          </Button>
-          <Button
-            variant="primary"
-            className="float-end"
-            disabled={
-              this.state.pNumber === null ||
-              (session && session.is_scheduled_beamline === false) ||
-              session.is_scheduled_time === false
-            }
-            onClick={this.handleSelectProposal}
+  return (
+    <Modal
+      show={show}
+      backdrop={showProposalsForm || 'static'}
+      onHide={() => {
+        if (showProposalsForm) {
+          handleHide();
+        }
```

Comment:
This condition does the opposite: it makes the dialog truly modal when shown because no proposal is selected. It does so by preventing the dialog from being dismissed with `Escape` in this situation.